astropy/io/fits/connect.py

```diff
@@ -91,6 +92,11 @@ def read_table_fits(input, hdu=None, astropy_native=False):
     astropy_native : bool, optional
         Read in FITS columns as native astropy objects where possible instead
         of standard Table Column objects. Default is False.
+    include_columns : list of str or None, optional
+        A list of column names to include in the output table. If None, include
+        *all* columns.  Note that this is best used for performance on a table
```

Comment:
Is there a severe performance penalty for excluding just a few columns?  Anyway, people may have different reasons for doing this, and for instance you may have a table where one column is an image with dims (2048, 2048) for each element and the user wants to exclude it.  (Which is a good reason to also have the complementary `exclude_names` arg.)